src/ui/actions/comments.ts

```diff
@@ -15,42 +15,28 @@ import type { ThreadFront as ThreadFrontType } from "protocol/thread";
 import { waitForTime } from "protocol/utils";
 import { Action } from "redux";
 import { RequestSummary } from "ui/components/NetworkMonitor/utils";
+import { ADD_COMMENT_MUTATION } from "ui/hooks/comments/useAddComment";
 import { selectors } from "ui/reducers";
 import { getCurrentTime } from "ui/reducers/timeline";
-import { PendingComment, Comment, Reply, SourceLocation, CommentOptions } from "ui/state/comments";
-import { User } from "ui/types";
+import { Comment, Reply, SourceLocation, CommentOptions } from "ui/state/comments";
+import { trackEvent } from "ui/utils/telemetry";
+import { mutate } from "ui/utils/apolloClient";
 
-import { getLoadedRegions } from "./app";
 import type { UIThunkAction } from "./index";
 import { setSelectedPrimaryPanel } from "./layout";
 import { seek } from "./timeline";
 
-type SetPendingComment = Action<"set_pending_comment"> & { comment: PendingComment | null };
 type SetHoveredComment = Action<"set_hovered_comment"> & { comment: any };
-type UpdatePendingCommentContent = Action<"update_pending_comment_content"> & { content: string };
 
-export type CommentsAction = SetPendingComment | SetHoveredComment | UpdatePendingCommentContent;
-
-export function setPendingComment(comment: PendingComment): SetPendingComment {
-  return { type: "set_pending_comment", comment };
-}
+export type CommentsAction = SetHoveredComment;
```

Comment:
IDK if this type is justifying its own existence anymore.

Reply:
Ah, screw it. Might as well do that one last thing– 179d707